contracts/Coordinator.sol

```diff
@@ -79,11 +82,18 @@ contract Coordinator is ICoordinator {
     function withdrawCollateralUnderNFT(uint256 amount, uint256 nftId) external override notImplementedYet {}
 
     function borrowUnderNFT(uint256 _nftId, uint256 _amount) external override {
-        IERC20(_tokenLvUSD).transfer(_tokenVaultOUSD, _amount);
+        IERC20(_tokenLvUSD).transfer(_tokenExchanger, _amount);
         CDPosition(_tokenCDP).borrowLvUSDFromPosition(_nftId, _amount);
     }
 
-    function repayUnderNFT(uint256 _nftId, uint256 _amount) external override notImplementedYet {}
+    function repayUnderNFT(uint256 _nftId, uint256 _amount) external override {
+        require(
+            CDPosition(_tokenCDP).getLvUSDBorrowed(_nftId) >= _amount,
```

Comment:
Don't we expect _amount to be equal or larger than CDPosition(_tokenCDP).getLvUSDBorrowed(_nftId) ?
Looks like the sign is flipped ?

Reply:
We cant repay more then we already borrowed. Checking if the amount of lvUSD under position is greater then what we attempt to repay and reverting otherwise